tests/testdata/nairobi.test.json

```diff
@@ -0,0 +1,4 @@
+{
+	"accountId": "f734f5aa37d06d86c3979f829f1394ab332ec83dcae287caf1c801dc33457318",
```

Comment:
@dxphilo can you omit the values for the test account? Maybe just rename it to testaccount.test.json and leave the values blank. Just leave a comment "Insert accountId and privateKey here"

Reply:
The reason is for open source security, don't want someone using this to go crazy with it

Reply:
But we would want the private key in order to run the UI test, 

I agree with jack here though, maybe it's better that an error is thrown if there's no file in the specified location, and add the location to `.gitignore`